test/verify.test.js

```diff
@@ -307,77 +309,129 @@ test.serial('Verify "assets" is an Array of Object with a glob Arrays in path pr
   t.true(gitlab.isDone());
 });
 
-test('Throw SemanticReleaseError if "assets" option is not a String or an Array of Objects', async t => {
+test.serial('Throw SemanticReleaseError if "assets" option is not a String or an Array of Objects', async t => {
+  const owner = 'test_user';
+  const repo = 'test_repo';
   const env = {GITLAB_TOKEN: 'gitlab_token'};
   const assets = 42;
+  const gitlab = authenticate(env)
+    .get(`/projects/${owner}%2F${repo}`)
+    .reply(200, {permissions: {project_access: {access_level: 40}}});
 
-  const [error] = await t.throwsAsync(
+  const aggregateError = await t.throwsAsync(
     verify(
       {assets},
-      {env, options: {repositoryUrl: 'https://gitlab.com/semantic-release/gitlab.git'}, logger: t.context.logger}
-    )
+      {env, options: {repositoryUrl: `https://gitlab.com/${owner}/${repo}.git`}, logger: t.context.logger}
+    ),
+    AggregateError,
+    'Error thrown but of unexpected type.'
   );
+  const errors = [...aggregateError];
```

Comment:
The intermediate `aggregateError` seems unnecessary. Why not just get the `errors` directly?

Reply:
Because it is less readable. Adding the intermediate helps understanding the code better.

Reply:
I'm sorry but I don't see how making the reader go through more hops to understand something is better in any way...

Reply:
Just to make sure I get it right.

You prefer this:
```
  const errors = [
    ...(await t.throwsAsync(
      verify(
        {assets},
        {env, options: {repositoryUrl: `https://gitlab.com/${owner}/${repo}.git`}, logger: t.context.logger}
      ),
      AggregateError,
      'Error thrown but of unexpected type.'
    )),
  ];

```  

Over this?:
```
  const aggregateError = await t.throwsAsync(
    verify(
      {assets},
      {env, options: {repositoryUrl: `https://gitlab.com/${owner}/${repo}.git`}, logger: t.context.logger}
    ),
    AggregateError,
    'Error thrown but of unexpected type.'
  );
  const errors = [...aggregateError];
```